bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/UniFiController.java

```diff
@@ -67,12 +67,19 @@
 
     private final Gson gson;
 
-    public UniFiController(HttpClient httpClient, String host, int port, String username, String password) {
+    private Boolean unifiOS;
```

Comment:
Please use the same naming convention as you do in the `UniFiControllerThingConfig`

Reply:
And make it `final` and place it below the `password` field.